tests/components/isy994/test_system_health.py

```diff
@@ -31,15 +31,16 @@ async def test_system_health(hass, aioclient_mock):
         unique_id=MOCK_UUID,
     ).add_to_hass(hass)
 
-    hass.data[DOMAIN] = {}
-    hass.data[DOMAIN][MOCK_ENTRY_ID] = {}
-    hass.data[DOMAIN][MOCK_ENTRY_ID][ISY_ROOT] = Mock(
-        connected=True,
-        websocket=Mock(
-            last_heartbeat=MOCK_HEARTBEAT,
-            status=MOCK_CONNECTED,
-        ),
+    isy_data = Mock(
+        root=Mock(
+            connected=True,
+            websocket=Mock(
+                last_heartbeat=MOCK_HEARTBEAT,
+                status=MOCK_CONNECTED,
+            ),
+        )
     )
+    hass.data[DOMAIN] = {MOCK_ENTRY_ID: isy_data}
```

Comment:
This is existing, but we aren't supposed to access `hass.data` in tests since its interacting with the integration internal details: https://developers.home-assistant.io/docs/development_testing?_highlight=writing&_highlight=te#writing-tests-for-integrations

Reply:
Open to suggestions on how to get around that--I couldn't find an easy way to patch around it.

Reply:
In a future PR, mock out the `pyisy` functions, and setup the integration using `await hass.config_entries.async_setup(entry.entry_id)`